src/app/reducers/api-response.ts

```diff
@@ -87,12 +87,17 @@ export function reducer(state: State = initialState, action: api.Actions | pagin
 			let hashtags = Array.from(new Set(tagStrings)).map(tag => {
 					return { tag, count: tagStrings.filter(y => y === tag).length };
 			});
+			let oldapiaggregations = state.aggregations;
+			let newapiaggregations = apiResponse.aggregations;
+			if (newapiaggregations === null) {
+				newapiaggregations = oldapiaggregations;
+			}
 
 			return Object.assign({}, state, {
 				pages:	[...state.pages, apiResponse],
 				entities: [...state.entities, ...apiResponse.statuses],
 				hashtags: [...hashtags],
-				aggregations: apiResponse.aggregations,
+				aggregations: newapiaggregations,
```

Comment:
Ohh sorry my mistake, It should be `apiResponse.aggregations || state.aggregation`, single `|` is bitwise operator(applicable on type numbers) thus the error.